packages/common/src/services/audius-backend/solana.ts

```diff
@@ -101,16 +102,22 @@ export const getRecentBlockhash = async (
 export const getTokenAccountInfo = async (
   audiusBackendInstance: AudiusBackend,
   {
+    tokenAccount,
     mint = DEFAULT_MINT,
-    tokenAccount
+    commitment
```

Comment:
You used a default in the hook, but not here? Seems like it should be consistent.

Reply:
ye